client/informers/externalversions/core/v1alpha1/interface.go

```diff
@@ -24,6 +24,8 @@ import (
 type Interface interface {
 	// MeshFederations returns a MeshFederationInformer.
 	MeshFederations() MeshFederationInformer
+	// ServiceExportses returns a ServiceExportsInformer.
+	ServiceExportses() ServiceExportsInformer
```

Comment:
This name is incorrect.  It should be `ServiceExports`.  Check maistra.mk for the cli option that fixes the name mapping.  I don't know if there's a tag that can be used to configure this (which would be easier).

Reply:
Thanks. I had to deal with this change in my wip for maistra/istio. I thought that was a typo. Fixed. English is hard :)
I couldn't find a tag for it, so I'm just copying the cli args from maistra.mk.